test/feature/steps/init.js

```diff
@@ -7,65 +7,62 @@ const sinonChai = require('sinon-chai');
 const path = require('path');
 const figlet = require('figlet');
 const chalk = require('chalk');
+const inquirer = require('inquirer');
 const controller = require('../../../lib/controller');
+const initInput = require('../../../lib/cli/input/init');
 const preferenceManager = require('../../../lib/utils/preference-manager');
 
 chai.use(sinonChai);
 chai.should();
 
-Before(() => {
-  global.logSpy = sinon.stub(console, 'log');
-});
-
-After(() => {
-  global.logSpy.restore();
-});
+global.promptStub = sinon.stub(inquirer, 'prompt');
+global.logSpy = sinon.stub(console, 'log');
 
 Given('a valid username as {string} and corresponding password as {string}', (user, pass) => {
   username = user;
   password = pass;
 });
 
-When('I run init command with {string} as username and {string} as password using {string}', (username, password, inputType, done) => {
+When('I run init command with {string} as username and {string} as password using {string}', async (username, password, inputType) => {
   if(inputType === 'flags') {
-      exec(`autolabjs init -u ${username} -p ${password}`, (err, stdout, stderr) => {
-        errorOutput = stdout;
-        done();
-      });
+    process.argv = ['/usr/local/nodejs/bin/node',
+      '/usr/local/nodejs/bin/autolab', 'init', '-u', username, '-p', password];
```

Comment:
autolabjs, not autolab. The same comment is applicable in all other feature tests as well.

By any chance, are we publishing the npm package as autolab, not autolabjs?